index.js

```diff
@@ -153,6 +155,10 @@ class ServerlessWebpack {
         .then(this.prepareOfflineInvoke)
         .then(this.wpwatch),
 
+      'before:step-functions-offline:start': () => BbPromise.bind(this)
+         .then(this.validate)
```

Comment:
You already spawn the `webpack:validate` in the prepareStepOfflineInvoke function, so you can remove the line here.

Reply:
ahh, you're right.
I missed that